src/com/mambu/apisdk/services/DatabaseService.java

```diff
@@ -28,6 +30,10 @@
 	private final static ApiDefinition createDatabaseBackup = new ApiDefinition(ApiType.CREATE_JSON_ENTITY,
 			DatabaseBackupRequest.class, DatabaseBackupResponse.class);
 
+	// Download DB backup API definition
+	private final static ApiDefinition downloadLatetDbBackupDefinition = new ApiDefinition(ApiType.GET_ENTITY,
```

Comment:
In order to be consistent with the other ApiDefinition within this class and also with the other definitions from the other services, I think we can remove the "Definition" word from the constant' name. Also, please note that "Latest" is misspelled.